src/tokens.ts

```diff
@@ -79,13 +79,15 @@ export namespace IDebugger {
      * Stop a running debug session.
      */
     stop(): Promise<void>;
+
```

Comment:
We could add a docstring here (to stay consistent with the rest of the interface):

```suggestion
    /**
     * Restore the state of a running debug session.
     */
```